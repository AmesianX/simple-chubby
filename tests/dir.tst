# try to create some files/dir
fileOpen /test/a 8
# returned FdId: 0
fileOpen /test 8
# returned FdId: 1
fileOpen /test/a 8
# returned FdId: 0
fileOpen /dir 4
# returned FdId: 2
fileOpen /dir/a 8
# returned FdId: 3
fileOpen /dir/b 8
# returned FdId: 4

# try to delete some files/dir
get 2
fileDelete 2
fileDelete 3
get 2
fileDelete 4
get 2
fileDelete 2

exit