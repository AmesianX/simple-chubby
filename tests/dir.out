0
1
0
2
3
4
|/dir/a|/dir/b
Failed
Deleted
/dir/b
Deleted

Deleted
